README.md

```diff
@@ -0,0 +1,79 @@
+gceimgutils
+===========
+
+A collection of utilities for image management in Google Compute Engine
+
+While the functionality provided by the utilities is more or less a duplication
+of functionality available with the gcloud CLI tools from the [google-cloud-sdk](https://cloud.google.com/sdk) the google-cloud-sdk does not provide consumable sources for packaging in a distribution nor is there a project for upstream contributions. This project focuses on image management only.
+
+All utilities expect the service account credentials in ~/.config/gce. The name of the credentials file is expected to match the project name followed by the .json extension.
```

Comment:
Does this mean the google-cloud-sdk is a requirement? If not, should we provide a sample config?

Reply:
No google-cloud-sdk is specifically not a requirement, as stated, google-cloud-sdk is not packagable. If it were this would not really be necessary.

Reply:
Are you going to provide a sample ~/.config/gce.?

Reply:
No example, it's the service-account file. Obviously I need to actually write that and not think I wrote it.